templates/base.html.twig

```diff
@@ -9,9 +9,10 @@
         {% include "@SuluWebsite/Extension/seo.html.twig" with {
             "seo": extension.seo|default([]),
             "content": content|default([]),
-            "urls": urls|default([]),
+            "localizations": localizations|default([]),
             "shadowBaseLocale": shadowBaseLocale|default(),
-            "defaultLocale": app.request.locale
+            "defaultLocale": app.request.locale,
+            "urls": urls|default([])
```

Comment:
Think this case can be removed as localizations is defined urls is not longer accessed when the variable is correct above?